client/lib/analytics/utils/signup_domain_origin.js

```diff
@@ -0,0 +1,8 @@
+const SIGNUP_DOMAIN_ORIGIN = {
```

Comment:
I just came up with these strings as POC, I'm open to changing them to fit our tracks props convention

Reply:
These seem fine to me. They're clear, and leaves no ambiguity to the domain origin. 👍  We can change them if requested later.